packages-content-model/roosterjs-content-model-core/lib/publicApi/domUtils/getCurrentSegmentTextFormat.ts

```diff
@@ -0,0 +1,29 @@
+import type { ContentModelSegment, ContentModelSegmentFormat } from 'roosterjs-content-model-types';
+
+/**
+ * Get the text format of the current selected segment
+ * @param currentSegment The segment to get format from
+ * @returns
+ */
+export function getCurrentSegmentTextFormat(
```

Comment:
We are actually getting format from a given segment, so you can remove the word "Current" from function name and parameter name.